docs/conf/portland/2020/lightning-talks.rst

```diff
@@ -3,6 +3,82 @@
 Lightning Talks
 ===============
 
-.. _speaker-lightning-talks-2020/eu:
+.. _speaker-lightning-talks-2020/na:
```

Comment:
Any specific reason this is hard-coded into the topic? I thought we specifically moved this type of content to include files..

Reply:
Yea, I don't quite understand why this was pulled in?

Reply:
Yup there was. 

A better way might have been to make an online include, but I didn't really have the time and patience to mess around with it.